config/routes.rb

```diff
@@ -1,4 +1,7 @@
 Rails.application.routes.draw do
+  resources :users, only: [:index, :show] do
+    resources :posts, only: [:index, :show]
+  end
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Defines the root path route ("/")
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/224763992-df8e5b9a-6fa4-4de0-b3b9-127961671bb7.png)

- [ ] It would be nice if you add a message such as **If you like this project then don't forget to give a star :star: on this repository** to make your readme file descriptive and professional :blush: